contrib/endpoints/src/api_manager/context/service_context.h

```diff
@@ -67,7 +67,10 @@ class ServiceContext {
 
   bool IsRulesCheckEnabled() const {
     return RequireAuth() && service().apis_size() > 0 &&
-           !config_->GetFirebaseServer().empty();
+        (!config_->GetFirebaseServer().empty() ||
```

Comment:
I think the intention is to separate the logic of server config and service config. 

liminw@, what do you think?